### PR TITLE
Improve gRPC migration doc

### PR DESCRIPTION
**Pull request by JamesNK**
Internal preview: https://review.docs.microsoft.com/en-us/aspnet/core/grpc/migration?view=aspnetcore-3.0&branch=pr-en-us-24749

### CODE REVIEW

File: aspnetcore/grpc/migration.md [hunk 1]
```diff
@@ -15,6 +15,21 @@ Due to the implementation of the underlying stack, not all features work in the
 > [!IMPORTANT]
 > gRPC C-core is in maintenance mode and [will be deprecated in favor of gRPC for .NET](https://grpc.io/blog/grpc-csharp-future/). gRPC C-core is not recommended for new apps.
 
+## Configure server and channel
+
+NuGet packages, configuration and startup code must be modified when migrating from gRPC C-Core to gRPC for .NET.
```
Comment (guardrex) [inline aspnetcore/grpc/migration.md:20]:
```suggestion
NuGet packages, configuration, and startup code must be modified when migrating from gRPC C-Core to gRPC for .NET.
```

File: aspnetcore/grpc/migration.md [hunk 2]
```diff
@@ -15,6 +15,21 @@ Due to the implementation of the underlying stack, not all features work in the
...
+
+gRPC for .NET has seprate NuGet packages for its client and server. The packages added depend upon whether an app is hosting gRPC services or calling them:
```
Comment (guardrex) [inline aspnetcore/grpc/migration.md:22]:
```suggestion
gRPC for .NET has separate NuGet packages for its client and server. The packages added depend upon whether an app is hosting gRPC services or calling them:
```

File: aspnetcore/grpc/migration.md [hunk 3]
```diff
@@ -15,6 +15,21 @@ Due to the implementation of the underlying stack, not all features work in the
...
+
+* [**Grpc.AspNetCore**](https://www.nuget.org/packages/Grpc.AspNetCore): gRPC services are hosted by ASP.NET Core. For information about configuring gRPC with ASP.NET Core, see <xref:grpc/aspnetcore>.
```
Comment (guardrex) [inline aspnetcore/grpc/migration.md]:
```suggestion
* [**`Grpc.AspNetCore`**](https://www.nuget.org/packages/Grpc.AspNetCore): gRPC services are hosted by ASP.NET Core. For information about configuring gRPC with ASP.NET Core, see <xref:grpc/aspnetcore>.
```

File: aspnetcore/grpc/migration.md [hunk 4]
```diff
@@ -15,6 +15,21 @@ Due to the implementation of the underlying stack, not all features work in the
...
+* [**Grpc.Net.Client**](https://www.nuget.org/packages/Grpc.Net.Client): gRPC clients use `GrpcChannel`, which internally uses networking functionality built into .NET. For information about configuring `GrpcChannel`, see <xref:grpc/client>.
```
Comment (guardrex) [inline aspnetcore/grpc/migration.md]:
```suggestion
* [**`Grpc.Net.Client`**](https://www.nuget.org/packages/Grpc.Net.Client): gRPC clients use `GrpcChannel`, which internally uses networking functionality built into .NET. For information about configuring `GrpcChannel`, see <xref:grpc/client>.
```

File: aspnetcore/grpc/migration.md [hunk 5]
```diff
@@ -15,6 +15,21 @@ Due to the implementation of the underlying stack, not all features work in the
...
 > gRPC C-core is in maintenance mode and [will be deprecated in favor of gRPC for .NET](https://grpc.io/blog/grpc-csharp-future/). gRPC C-core is not recommended for new apps.
 
+## Configure server and channel
+
+NuGet packages, configuration and startup code must be modified when migrating from gRPC C-Core to gRPC for .NET.
+
+gRPC for .NET has seprate NuGet packages for its client and server. The packages added depend upon whether an app is hosting gRPC services or calling them:
+
+* [**Grpc.AspNetCore**](https://www.nuget.org/packages/Grpc.AspNetCore): Services are hosted by ASP.NET Core. For information about configuring gRPC with ASP.NET Core, see <xref:grpc/aspnetcore>.
+* [**Grpc.Net.Client**](https://www.nuget.org/packages/Grpc.Net.Client): Clients use `GrpcChannel`, which internally uses networking functionality built into .NET. For information about configuring `GrpcChannel`, see <xref:grpc/client>.
```
Comment (guardrex) [inline aspnetcore/grpc/migration.md:25]:
```suggestion
* [**`Grpc.Net.Client`**](https://www.nuget.org/packages/Grpc.Net.Client): Clients use `GrpcChannel`, which internally uses networking functionality built into .NET. For information about configuring `GrpcChannel`, see <xref:grpc/client>.
```

File: aspnetcore/grpc/migration.md [hunk 6]
```diff
@@ -15,6 +15,21 @@ Due to the implementation of the underlying stack, not all features work in the
 > [!IMPORTANT]
 > gRPC C-core is in maintenance mode and [will be deprecated in favor of gRPC for .NET](https://grpc.io/blog/grpc-csharp-future/). gRPC C-core is not recommended for new apps.
 
+## Configure server and channel
+
+NuGet packages, configuration and startup code must be modified when migrating from gRPC C-Core to gRPC for .NET.
+
+gRPC for .NET has seprate NuGet packages for its client and server. The packages added depend upon whether an app is hosting gRPC services or calling them:
+
+* [**Grpc.AspNetCore**](https://www.nuget.org/packages/Grpc.AspNetCore): Services are hosted by ASP.NET Core. For information about configuring gRPC with ASP.NET Core, see <xref:grpc/aspnetcore>.
```
Comment (guardrex) [inline aspnetcore/grpc/migration.md:24]:
```suggestion
* [**`Grpc.AspNetCore`**](https://www.nuget.org/packages/Grpc.AspNetCore): Services are hosted by ASP.NET Core. For information about configuring gRPC with ASP.NET Core, see <xref:grpc/aspnetcore>.
```

File: aspnetcore/grpc/migration.md [hunk 7]
```diff
@@ -15,6 +15,21 @@ Due to the implementation of the underlying stack, not all features work in the
...
+* [**Grpc.Net.Client**](https://www.nuget.org/packages/Grpc.Net.Client): Clients use `GrpcChannel`, which internally uses networking functionality built into .NET. For information about configuring `GrpcChannel`, see <xref:grpc/client>.
+
+When migration is complete the `Grpc.Core` package should be removed from the app. `Grpc.Core` contains large native binaries and removing the package reduces app size.
```
Comment (guardrex) [inline aspnetcore/grpc/migration.md:27]:
```suggestion
When migration is complete, the `Grpc.Core` package should be removed from the app. `Grpc.Core` contains large native binaries, and removing the package reduces app size.
```

File: aspnetcore/grpc/migration.md [hunk 8]
```diff
@@ -15,6 +15,21 @@ Due to the implementation of the underlying stack, not all features work in the
...
+
+## Code generated services and clients
+
+gRPC C-Core and gRPC for .NET share many APIs, and code generated from `.proto` files is compatible with both gRPC implementions. Most clients and service can be migrated from C-Core to gRPC for .NET without changes.
```
Comment (guardrex) [inline aspnetcore/grpc/migration.md:31]:
```suggestion
gRPC C-Core and gRPC for .NET share many APIs, and code generated from `.proto` files is compatible with both gRPC implementations. Most clients and service can be migrated from C-Core to gRPC for .NET without changes.
```

**Review by guardrex — APPROVED** [review]
Just nits.

File: aspnetcore/grpc/migration.md [hunk 9]
```diff
@@ -15,6 +15,21 @@ Due to the implementation of the underlying stack, not all features work in the
...
 > gRPC C-core is in maintenance mode and [will be deprecated in favor of gRPC for .NET](https://grpc.io/blog/grpc-csharp-future/). gRPC C-core is not recommended for new apps.
 
+## Configure server and channel
+
+NuGet packages, configuration, and startup code must be modified when migrating from gRPC C-Core to gRPC for .NET.
+
+gRPC for .NET has separate NuGet packages for its client and server. The packages added depend upon whether an app is hosting gRPC services or calling them:
+
+* [**`Grpc.AspNetCore`**](https://www.nuget.org/packages/Grpc.AspNetCore): Services are hosted by ASP.NET Core. For information about configuring gRPC with ASP.NET Core, see <xref:grpc/aspnetcore>.
+* [**Grpc.Net.Client**](https://www.nuget.org/packages/Grpc.Net.Client): Clients use `GrpcChannel`, which internally uses networking functionality built into .NET. For information about configuring `GrpcChannel`, see <xref:grpc/client>.
```
Comment (JamesNK) [inline aspnetcore/grpc/migration.md:25]:
```suggestion
* [**`Grpc.Net.Client`**](https://www.nuget.org/packages/Grpc.Net.Client): Clients use `GrpcChannel`, which internally uses networking functionality built into .NET. For information about configuring `GrpcChannel`, see <xref:grpc/client>.
```

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**